features/net-http/handled.feature

```diff
@@ -16,7 +16,7 @@ Scenario: A handled error sends a report
   And the event "unhandled" is false for request 0
   And the event "severity" equals "warning" for request 0
   And the event "severityReason.type" equals "handledError" for request 0
-  And the exception "errorClass" ends with "s.PathError" for request 0
+  And the exception is a PathError
```

Comment:
It might be worth including "for request 0" in this step so that it's clear what it operates on